src/UserNamespaces.jl

```diff
@@ -162,11 +162,15 @@ function build_executor_command(exe::UserNamespacesExecutor, config::SandboxConf
         append!(cmd_string, ["--entrypoint", config.entrypoint])
     end
 
+    needs_userxattr = exe isa UnprivilegedUserNamespacesExecutor && !Sandbox.probe_xattr("user")
```

Comment:
I'm waffling on whether this is the best way to test this.  I'm thinking it might be better to just add some C code to create a little probe that does things inside of sandboxes.